spring-amqp/src/main/java/org/springframework/amqp/support/converter/SerializerMessageConverter.java

```diff
@@ -48,6 +60,10 @@
 
 	private volatile boolean ignoreContentType = false;
 
+	private volatile ClassLoader defaultDeserializerClassLoader;
```

Comment:
Maybe we should make some comment that defaults to the `defaultClassLoader`. 
See `ClassUtils.forName()`.

My observation:
1. `new DefaultDeserializer()` makes it without `CL`.
2. The injected deserializer (`setDeserializer()`) maybe without `CL` as well.
3. `ConfigurableObjectInputStream` delegates to the `ClassUtils.forName()`.